com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -249,7 +251,7 @@ public UniversalRenderer(UniversalRendererData data) : base(data)
             }
             m_OnRenderObjectCallbackPass = new InvokeOnRenderObjectCallbackPass(RenderPassEvent.BeforeRenderingPostProcessing);
 
-            m_PostProcessPasses = new PostProcessPasses(data.postProcessData, m_BlitMaterial);
+            m_PostProcessPasses = new PostProcessPasses(data.postProcessData, m_CoreBlitMaterial);
```

Comment:
Passing along this (potentially) custom shader for blit might need to be revisited.
It is public but there is no UI option to change it.
Some of our tests might be touching this.

Reply:
In any case, our blit material is set directly on Blitter during configuration.
The split between coreblit and blit should also be revisited.

Reply:
What uses our Blit material after this change?
I think there should be only CoreBlit. 
And maybe one or two feature specific special cases as their own shaders if required.

Reply:
Should we merge `m_CoreBlitMaterial` and `m_BlitMaterial` ?

As for the access to the material, there's a path to that through the blitter API IIRC. 
Need to figure out which to promote instead of having two paths to the material.